packages/app-desktop/gui/ItemList.tsx

```diff
@@ -76,6 +78,10 @@ class ItemList extends React.Component<Props, State> {
 		if (this.props.onKeyDown) this.props.onKeyDown(event);
 	}
 
+	onDrop(event: any) {
+		if (this.props.onNoteDrop) this.props.onNoteDrop(event);
```

Comment:
This ItemList component is generic, so please could you rename props.onNoteDrop to props.onDrop?